proto/dymensionxyz/dymension/rollapp/metadata.proto

```diff
@@ -24,8 +24,12 @@ message RollappMetadata {
   string token_symbol = 9;
   // explorer_url is the rollapp's block explorer link
   string explorer_url = 10;
-  // fee_base_denom is the base denom for fees
-  string fee_base_denom = 11;
-  // native_base_denom is the base denom for the native token
-  string native_base_denom = 12;
+  // fee_denom is the base denom for fees
+  DenomMetadata fee_denom = 11;
```

Comment:
@omritoptix won't it be the same as the `native_denom`? we need additional struct for `fee_denom`?
IIRC we expect the denoms to be registered in the bank model for UI support

Reply:
not necesarily. we may have rollapps which will have gov denom and operate on e.g udsc as fee etc.